external/kafka-0-10-sql/src/main/scala/org/apache/spark/sql/kafka010/KafkaOffsetReader.scala

```diff
@@ -563,12 +526,6 @@ private[kafka010] class KafkaOffsetReader(
         && !Thread.currentThread().isInterrupted) {
         Thread.currentThread match {
           case ut: UninterruptibleThread =>
-            // "KafkaConsumer.poll" may hang forever if the thread is interrupted (E.g., the query
-            // is stopped)(KAFKA-1894). Hence, we just make sure we don't interrupt it.
-            //
-            // If the broker addresses are wrong, or Kafka cluster is down, "KafkaConsumer.poll" may
-            // hang forever as well. This cannot be resolved in KafkaSource until Kafka fixes the
-            // issue.
```

Comment:
Hmm, should we leave comment for this too? At least existing comment explains why we need uninterruptible behavior here. Removing this makes us hard to understand the reason in the future.

Reply:
There is an agreement to remove uninterruptible thread usage here in the next commit.

Reply:
Ah, I see. Good to know.